beancount_import/matching.py

```diff
@@ -240,6 +258,28 @@ def get_fuzzy_date_range(self, orig_date: datetime.date):
                                 self.fuzzy_match_days + 1):
             yield orig_date + datetime.timedelta(days=day_offset)
 
+    def fuzz_date_currency_key(self, key: DateCurrencyKey) -> Iterable[DateCurrencyKey]:
+        for d in self.get_fuzzy_date_range(key[0]):
+            yield d, key[1]
+
+    def _date_currency_key(self, entry: Transaction, mp: MatchablePosting) -> DateCurrencyKey:
+        pw = get_posting_weight(mp.posting)
+        currency = ""
```

Comment:
Is it possible to have a missing currency for a matchable posting?

Reply:
I have no idea. I don't understand the loader code well enough to know.